### DISCUSSION TITLE
some locations aren't showing up when i try to install them, and im not sure how to fix it

### DISCUSSION THREAD

**Discussion by WizardSteak107** [Monika-After-Story/MonikaModDev#9469] [QA - tech support + general]
I have mutliple locations installed but i am unable to go to some of them. ive tried to install new ones multiple times, but they still don't show up. can someone help with this?

**Comment by Iceninja64** [comment]
You probably don't have the affection to change rooms yet

**Comment by WizardSteak107** [comment]
I can go to other rooms though. Just not all the ones that I have installed
